src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Cmdlets/Policy/Edit-AzRecoveryServicesBackupSchedulePolicyClientObject.ps1

```diff
@@ -48,6 +48,10 @@
 
         [Parameter(ParameterSetName='ModifySchedulePolicy', HelpMessage='Specifies the days of the week for differential backup.')]
         [ValidateScript({
+            if ($_.Length -eq 0) {
+				throw "DifferentialRunDay cannot be an empty array."
+			}
+
```

Comment:
why do we need this ?

Reply:
Sir it's because of what Vincent dai said in the azure PR. https://github.com/Azure/azure-powershell/pull/21925#discussion_r1242430712